### DISCUSSION TITLE
Data capture integrations

### DISCUSSION THREAD

**Discussion by oddieis** [twentyhq/twenty#6799] [Engineering vision]
It would be very helpful to remove friction from data capture by enabling one click browser plugins a la Folk CRM, so with a single click I can pull in linkedin records, create records from my browser url, etc. Are there any plans?

**Comment by BOHEUS** [comment]
There is Chrome extension for Twenty (https://chromewebstore.google.com/detail/twenty/lanlmmgcjjmlbfbooodedgblillmnafb), although it's not updated from what I can see, if you have any ideas about new functionality, just submit issues

**Reply by oddieis** [reply]
This is a good start. It would be good to add the ability to integrate the chrome plugin to gmail, to twitter, and to any wb page to be added to the crm. and i have not checked but it would be ideal that the plugin allows integration with a self hosted instance of twenty.

**Reply by FelixMalfait** [reply]
Yes we decided to focus on the core project for now, we'll bring back the Chrome extension in the future at some point for sure
The Chrome extension was designed to work with self-hosted instance (you can input a server url)

**Comment by JhumanJ** [comment]
Hey, I created a small Linkedin extension for Twenty (compatible with self hosted instances). It's not on the chrome store but you can check it out and try it here if you're interested: https://github.com/JhumanJ/twenty-crm-extension. It's great for 1 click Linkedin capture.
<img width="1008" height="642" alt="image" src="https://github.com/user-attachments/assets/8f6cbd48-4abc-48b9-8b1d-225d537c2dd9" />

**Reply by neilk17** [reply]
Thanks to the previous contributor I built this based on his code:

https://github.com/neilk17/twenty-crm-chrome-extension

I will be adding the ability to add data from twitter and gmail too this week

**Comment by BOHEUS** [comment]
I'm closing this discussion in favor of collective discussion of integrations https://github.com/twentyhq/twenty/discussions/16735
